IPython/terminal/ipapp.py

```diff
@@ -335,9 +334,3 @@ def load_default_config(ipython_dir=None):
     app.config_file_paths.append(profile_dir)
     app.load_config_file()
     return app.config
-
-launch_new_instance = TerminalIPythonApp.launch_instance
-
-
-if __name__ == '__main__':
-    launch_new_instance()
```

Comment:
You need to keep the `launch_new_instance = TerminalIPythonApp.launch_instance`, it is imported in other places.

Reply:
Thanks for the report, it should be fixed now.